stable/nats/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: nats
-version: 0.0.3
+version: 0.0.4
```

Comment:
this needs another bump as some other merge has gone in first: https://github.com/helm/charts/blob/master/stable/nats/Chart.yaml